Userland/Demos/CatDog/SpeechBubble.cpp

```diff
@@ -5,9 +5,18 @@
  */
 
 #include "SpeechBubble.h"
+#include <AK/Random.h>
+#include <AK/StringView.h>
+#include <AK/Vector.h>
 #include <LibGUI/Painter.h>
 #include <LibGfx/Palette.h>
 
+static Vector<StringView> messages = {
```

Comment:
I think this can be a `static constexpr Array<StringView, 3>`

Reply:
Sure, done.